apps/docs/content/_partials/project_setup.mdx

```diff
@@ -29,14 +29,16 @@ Now we are going to set up the database schema. We can use the "User Management
 
 You can pull the database schema down to your local project by running the `db pull` command. Read the [local development docs](/docs/guides/cli/local-development#link-your-project) for detailed instructions.
 
-</Admonition>
+{/* TODO: discuss */}
```

Comment:
There are certain places where it feels like other content should be part of the admonition, but maybe there's a reason they aren't.

For example, here, as you scroll down, it looks like the code has nothing to do with the admonition.